srml/babe/src/lib.rs

```diff
@@ -133,11 +139,24 @@ decl_storage! {
 		/// The randomness under construction
 		UnderConstruction: [u8; VRF_OUTPUT_LENGTH];
 
-		/// The randomness for the next epoch
-		NextEpochRandomness: [u8; VRF_OUTPUT_LENGTH];
+		/// The data for the next epoch
+		NextEpoch get(next_epoch): Epoch;
 
 		/// The current epoch
 		EpochIndex get(epoch_index): u64;
+
+		/// The number of slots per epoch
+		SlotsPerEpoch get(slots_per_epoch) build(|config: &GenesisConfig| {
+			config.slots_per_epoch
```

Comment:
I think you can just do `SlotsPerEpoch get(slots_per_epoch) config(): u64;` instead.

Reply:
I was not aware of that.  Thanks!

Reply:
Is this value variable? Aka being changed per epoch?

Reply:
It doesn't make sense to have this variable without the `EpochChange` digest containing the `next_slot_number` (removed in this PR).

Reply:
@bkchr @rphmeier it is indeed not variable, as keeping it constant simplifies the code significantly.  I can make it variable later, but I would like to get this code merged first.